src/screens/WelcomeScreen/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { SafeAreaView, ScrollView, StatusBar, Text, View, useColorScheme } from 'react-native';
+import {
+  Colors,
+  DebugInstructions,
+  Header,
+  LearnMoreLinks,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+
+import styles from './styles';
+
+const Section: React.FC<{
```

Comment:
What's your viewpoint on declaring more than one React Component per file?

- Is it worth to move this to a separate file?
- Is it worth to do it now? Or maybe only when it's needed?